/*
 * ConfigInc.h
 *
 *  Created on: 2016-8-22
 *      Author: Ralf
 */

#ifndef CONFIGINC_H_
#define CONFIGINC_H_

#include "Demo.pb.h"
#include "ConfigBuilding.pb.h"
#include "CommonGiftConfig.pb.h"
#include "UserConfig.pb.h"
#include "ConfigItem.pb.h"
#include "ConfigProductLine.pb.h"

#endif /* CONFIGINC_H_ */
